client/src/components/Reviews/ReviewList.jsx

```diff
@@ -10,35 +10,14 @@ import Review from './Review.jsx'
 //base component
 function ReviewList(props) {
   const [ isLoading, setIsLoading ] = useState(true);
-  const [ reviews, setReviews ] = useState([]);
+  const [ reviews, setReviews ] = useState([props.reviews]);
   const [ productId, setProductId ] = useState(props.id);
```

Comment:
if you're taking in the reviews and id as props from app, do you need to create another useState hook?

Also, now that you are not calling getReviews from within this component, do you still need productId?

Reply:
true--I don't need setReviews since it's only getting passed down via props. I don't think I need setProductId either. 

I will, however, need setIsLoading for when I filter reviews by star rating.

Reply:
I think i'm a little confused on the isLoading part - you're taking in the isLoading state from the App, and you're also creating another isLoading state that is local to your ReviewList component, but so far it looks like you're passing down the isLoading state from the App. 

I think each component having its own loading state makes sense, but considering that the isLoading from the App is supposed to control the loading phase for the entire app to load, is it necessary to take that state down to a child component? Might be cleaner to decouple the logic and have the isLoading states independent for each component.

I'm also not sure how the filter review portion works, so perhaps I'm just missing the context!

Reply:
Review filtering doesn't work yet... I'm not sure whether I'll need to maintain isLoading somewhere else, but for now I can get rid of it.

Will make changes incorporating your comments.